<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationList</class>
 <widget class="QWidget" name="InformationList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>安卓消防教务系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>刷新页面</string>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
       <property name="icon">
        <iconset resource="../assets/resource.qrc">
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logoutBtn">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>刷新页面</string>
       </property>
       <property name="text">
        <string>注销</string>
       </property>
       <property name="icon">
        <iconset resource="../assets/resource.qrc">
         <normaloff>:/icons/logout.png</normaloff>:/icons/logout.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="batchBtn">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>刷新页面</string>
       </property>
       <property name="text">
        <string>批量处理</string>
       </property>
       <property name="icon">
        <iconset resource="../assets/resource.qrc">
         <normaloff>:/icons/batch.png</normaloff>:/icons/batch.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="SearchComboBox" name="cbx_clazz">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbx_recruiter">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbx_lecturer">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetFilterBtn">
       <property name="text">
        <string>重置</string>
       </property>
       <property name="icon">
        <iconset resource="../assets/resource.qrc">
         <normaloff>:/icons/reset.png</normaloff>:/icons/reset.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>信息脱敏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SwitchButton" name="sensitiveToggleBtn" native="true">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>请输入学员姓名/身份证号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchBtn">
         <property name="text">
          <string>搜索</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/assets/icons/search.png</normaloff>:/assets/icons/search.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="studentsDataTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>218</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>每页数量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_pageSize">
         <property name="currentText">
          <string>50</string>
         </property>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="prePageBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentPageLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>第1页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sumPageLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>共1页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextPageBtn">
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>components/SwitchButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchComboBox</class>
   <extends>QComboBox</extends>
   <header>components/searchcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets/resource.qrc"/>
 </resources>
 <connections/>
</ui>
